ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -71,14 +71,17 @@
           </template>
 
           <!-- Civic Address -->
-        <v-row no-gutters class="px-6 pt-1">
-          <v-col cols="3" class="pt-1">
-            <h3>Civic Address</h3>
-          </v-col>
-          <v-col cols="9" class="pt-1">
-            <p>{{ '' || '(Not Entered)' }}</p>
-          </v-col>
-        </v-row>
+          <template>
+            <v-row no-gutters class="px-6 pt-1">
+              <v-col cols="3" class="pt-1">
+                <h3>Civic Address</h3>
+              </v-col>
+              <v-col cols="9" class="pt-1">
+                <p>{{ getMhrRegistrationLocation.address.street || '(Not Entered)' }} {{ getMhrRegistrationLocation.address.streetAdditional || '' }}<br/>
+                  {{ getMhrRegistrationLocation.address.city || '(City Not Entered)' }} {{ getMhrRegistrationLocation.address.region=='British Columbia'?'BC': '(Province Not Entered)' }}</p>
```

Comment:
Some new lines here separating the templating might make this easier for the brain to parse :)